### PR TITLE
Add Semester Course Rollout and Collaborative Development Blog Post

### PR DESCRIPTION
**Pull request by brymz**
This blog is the result of a discussion with @ethanwhite, @tracykteal and myself following the recent blogs on Jabberwocky.weecology.org formally announcing the Data Carpentry Semester Course. The purpose is to generate some traffic from the DC Blog to Jabberwocky and the Semester Course site.

### CODE REVIEW

File: _posts/2016-11-28-course-collaboration.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+subheadline: "Curriculum"
+title: "Data Carpentry Expanding Collaborative Lesson Development to Semester-long Courses"
+teaser: "Let's build on Data Carpentry's core strength of collaborative lesson development."
+header:
```
Comment (tracykteal) [inline _posts/2016-11-28-course-collaboration.md]: Just suggesting a slight title revision.
title: "Collaborative Lesson Development to Semester-long Courses"
teaser: "Building on Data Carpentry's core strength of collaborative lesson development."

**Comment by brymz** [comment]
@ethanwhite Can/have you predetermined the link for the "Collaborative Course Development" post on Jabberwocky so that I can include it here?

**Comment by ethanwhite** [comment]
@brymz - not yet, but I'm pretty sure that once this gets merged it will go live (is that right Tracy?) so you'll need to wait until the 2nd post goes out anyway.

**Comment by brymz** [comment]
My understanding is that that date included in the file name determines when the post goes live. Right now, I have the general jabberwocky link set up to direct to the 2nd post. So, that would work until you make another post to jabberwocky and should give us plenty of time to update the link on my post.

**Comment by tracykteal** [comment]
Right, if I merge the PR, it goes live. I think this can go live before the bigger collaborative lesson development one and just not link to it. 

We'll link the other way from that post, to include this one.

**Comment by brymz** [comment]
@tracykteal, It seems our comments crossed in the cloud... So, I should remove the link to @ethanwhite's 2nd post for now? Can we put it back after his goes live?

**Comment by tracykteal** [comment]
I don't think github pays attention to the date, so once it's merged it goes live. If we want to delay posting, I would delay the merge.

I would say either, let's post today (and yes we can add the link to the next post later once it goes up), or integrate with a broader post and close this PR. @brymz Let me know what you want to do.

**Comment by brymz** [comment]
I have made the revisions to the `title:` and `teaser:` and a few minor edits. 

I think I would prefer to sync the release of this post with @ethanwhite's. I will update the link on Mon and send a note to merge the PR. 

Thanks for the opportunity to post on the DC blog!

**Comment by ethanwhite** [comment]
I'm happy with whatever in terms of order. Here's the link to my post, which will go live Monday at 9 a.m. ET:

http://jabberwocky.weecology.org/2016/11/28/the-potential-for-collaborative-open-lesson-development-for-college-coursework

**Pull request merged**